### ISSUE TITLE
Curio Slot Issue

### ISSUE THREAD

**Issue by TeeJayMac** [gigaherz/ToolBelt#133]
I am using the newest version of Tool Belt AND Curios API for 1.19.3 Forge as of typing this (**_curios-forge-1.19.3-5.1.4.0_** and **_ToolBelt-1.19.3-1.19.7_**) and it says the Tool Belt can be equipped in the belt slot, but when I open the curio slot menu, I only see the charm slot.

**Comment by gigaherz** [comment]
Oh apparently I forgot to re-enable curios support in the 1.19.3 version after Curios was ported.
I just noticed while porting to 1.19.4

**Comment by TeeJayMac** [comment]
Does Tool Belt work on 1.19.3 without curios? I made sure the keybinds weren't conflicting with anything and I couldn't swap tools. I also couldn't find a belt slot anywhere.

**Comment by gigaherz** [comment]
By default the integrated belt slot disables itself if curios is present, but you can change that in the config.
The key for it defaults to V but you can change it in the control options

**Comment by Artiow** [comment]
> Oh apparently I forgot to re-enable curios support in the 1.19.3 version after Curios was ported. I just noticed while porting to 1.19.4

So, does the latest ToolBelt-1.19.4-1.20.0 support the latest Curious version in the 1.19.4 game version?

**Comment by gigaherz** [comment]
> So, does the latest ToolBelt-1.19.4-1.20.0 support the latest Curious version in the 1.19.4 game version?

Yes. I have updated my development environment to latest Curios, JEI and Forge versions for 1.19.4, and it works fine with no changes, so the latest release should similarly work fine as-is.
